src/components/MovieDisplay.js

```diff
@@ -0,0 +1,19 @@
+const MovieDisplay = (props) => {
+    return (
+        <section className="movieSuggestion">
```

Comment:
Looks FANTASTIC! I think a stretch goal to consider CSS wise might be to have the image fit on the users screen.
maybe adding something like

HTML:
<div class="imgContainer">
<img src alt/>
</div>

CSS:

.imgContainer{
// trying to only be as tall/wide as x % of window
width: 30vw
height: 50vh

// setting constraints so it doesn't get out of hand hahah
max-width: 500px
max-height: 700px
}